includes/class-migration.php

```diff
@@ -514,15 +514,8 @@ public static function add_default_settings() {
 	 * Add the ActivityPub capability to all users that can publish posts.
 	 */
 	private static function add_activitypub_capability() {
-		// Get all WP_User objects that can publish posts.
-		$users = \get_users(
-			array(
-				'capability__in' => array( 'publish_posts' ),
-			)
-		);
-
-		// Add ActivityPub capability to all users that can publish posts.
-		foreach ( $users as $user ) {
+		// Add ActivityPub capability to all users.
+		foreach ( \get_users() as $user ) {
```

Comment:
Same here: Only users with the `publish_post` capability should be migrated, the rest has to be done by hand.